app/src/main/java/com/example/memeitupapp/ui/memeitup/MemeItUpActivity.kt

```diff
@@ -3,11 +3,33 @@ package com.example.memeitupapp.ui.memeitup
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.example.memeitupapp.R
+import com.example.memeitupapp.data.entity.Meme
+import com.example.memeitupapp.util.DEFAULT_INT
+import com.example.memeitupapp.util.Data
+import com.example.memeitupapp.util.Event
+import kotlinx.android.synthetic.main.activity_main.activity_main_call_service_button
+import kotlinx.android.synthetic.main.activity_main.activity_main_first_meme_id
 
 class MemeItUpActivity : AppCompatActivity() {
```

Comment:
here the same, in the viewmodel same, rename.

Reply:
renamed, thanks